gtsam_example.h

```diff
@@ -1,12 +1,10 @@
 /* ----------------------------------------------------------------------------
-
```

Comment:
I don't understand why this and some other files have been moved to the top level? Wouldn't it make more sense to have all this encapsulated in a project folder aka `example`?

Reply:
I discussed with Frank about a month ago and this is the result...